container_images.mk

```diff
@@ -0,0 +1,27 @@
+COREDNS_IMAGE=k8s.gcr.io/coredns@sha256:3e2be1cec87aca0b74b7668bbe8c02964a95a402e45ceb51b2252629d608d03a
+COREDNS_IMAGE_NAME=coredns
+COREDNS_IMAGE_VERSION=1.1.3
```

Comment:
It appears that `kubeadm` uses CoreDNS 1.2.2 when deploying K8s 1.11. Where did you get this version from?

Reply:
Huh no you're right, that's at least what it says in the ChangeLog. Odd.